common/sgx/endorsements.h

```diff
@@ -0,0 +1,114 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#ifndef _OE_COMMON_OE_ENDORSEMENTS_H
+#define _OE_COMMON_OE_ENDORSEMENTS_H
+
+#include <openenclave/bits/attestation.h>
+#include <openenclave/bits/defs.h>
+#include <openenclave/bits/result.h>
+
+OE_EXTERNC_BEGIN
+
+/*! \struct SGX endorsement item
+ */
+typedef struct _oe_sgx_endorsement_item
+{
+    uint8_t* data;
+    uint32_t size;
+} oe_sgx_endorsement_item;
+
+/*! \struct oe_sgx_endorsements
+ *
+ * \brief SGX endorsements structure
+ *
+ * The generic oe_endorsements_t structure is parsed and converted into this
+ * internal structure.  The order of the generic data elements should
+ * coincide with the order of the fields in this structure.
+ *
+ * Data format: All data comes from the Data Center Attestation Primitives(DCAP)
+ * Client.
+ *
+ * For Azure DCAP Client
+ * (https://github.com/microsoft/Azure-DCAP-Client/blob/master/src/dcap_provider.h)
+ * see **sgx_ql_revocation_info_t** and sgx_qe_identity_info_t.
+ *
+ * For Intel DCAP Client
+ * (https://github.com/intel/SGXDataCenterAttestationPrimitives/blob/master/README.md)
+ * see TBD.
```

Comment:
Is there an issue # filed to track filling in this TBD?

Reply:
Updated to have the intel structure name.  It was just updated yesterday!